packages/tpetra/core/ext/TpetraExt_MatrixMatrix_def.hpp

```diff
@@ -265,21 +268,19 @@ void Multiply(
     CrsWrapper_CrsMatrix<Scalar, LocalOrdinal, GlobalOrdinal, Node> crsmat(C);
 
     MMdetails::mult_A_B(Aview, Bview, crsmat, label,params);
+  }
 
 #ifdef HAVE_TPETRA_MMM_TIMINGS
-  }
```

Comment:
Also, having this closing brace inside a timing #ifdef didn't seem right, since when timing was enabled it matched with the "else {" on line 261, when I think it was supposed to match with line 248. This actually changes the semantics of the function depending on if timing is enabled.